packages/vm/lib/index.ts

```diff
@@ -135,7 +135,7 @@ export default class VM extends AsyncEventEmitter {
 
     if (opts.common) {
       //EIPs
-      const supportedEIPs = [2537]
+      const supportedEIPs = [2537, 2929]
```

Comment:
Can you also add a note on the support of this EIP in the code docs above (line 43, so annoying that GitHub doesn't allow comments on non-changed code parts)?

Reply:
Fixed in e922e1b